openhtf/io/output/upload_json.py

```diff
@@ -0,0 +1,72 @@
+"""Module for uploading JSON-formatted test record files to server."""
+
+import os
+import logging
+import requests
+
+from json import JSONEncoder
+from openhtf import conf
+from openhtf.io.output.json_factory import OutputToJSON
+
+conf.Declare('project', 'project name')
```

Comment:
Oh, I understand why you want the "target" here. the way to use it  is duo to the server entry point may have other info between data_server and project, if put them together, code needs to do some parsing to meet the requirement